contract-sdk/src/lib.rs

```diff
@@ -9,6 +9,7 @@ pub mod env;
 pub mod error;
 pub mod event;
 pub mod memory;
+pub mod modules;
```

Comment:
Oh I thought this would be added to `contract-sdk-types`?

Reply:
Yeah of course that makes more sense, thanks :-)